src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.models.Bulldozer;
+import core.basesyntax.models.Excavator;
+import core.basesyntax.models.Machine;
+import core.basesyntax.models.Truck;
+
 public class MainApp {
+    private static Machine excavator = new Excavator();
+    private static Machine bulldozer = new Bulldozer();
+    private static Machine truck = new Truck();
```

Comment:
let's better make these variables local please
class fields should describe the state of class instance, now you use these Machines only inside your method